projects/odf_transform/test/test_headers/CTD_HUD2018030_003_01_DN.ODF.json.nc.txt

```diff
@@ -0,0 +1,1231 @@
+netcdf CTD_HUD2018030_003_01_DN.ODF.json {
+dimensions:
+	z = 62 ;
+variables:
+	string filename ;
+		string filename:_FillValue = "nan" ;
+	string country ;
+		string country:_FillValue = "nan" ;
+	string cruise_id ;
+		string cruise_id:_FillValue = "nan" ;
+	string scientist ;
+		string scientist:_FillValue = "nan" ;
+	string platform ;
+		string platform:_FillValue = "nan" ;
+	string instrument_type ;
+		string instrument_type:_FillValue = "nan" ;
+	string instrument_serial_number ;
+		string instrument_serial_number:_FillValue = "nan" ;
+	float latitude ;
+		latitude:_FillValue = NaNf ;
+		latitude:long_name = "Latitude" ;
+		latitude:standard_name = "latitude" ;
+		latitude:units = "degrees_north" ;
+	float longitude ;
+		longitude:_FillValue = NaNf ;
+		longitude:long_name = "Longitude" ;
+		longitude:standard_name = "longitude" ;
+		longitude:units = "degrees_east" ;
+	string geographic_area ;
+		string geographic_area:_FillValue = "nan" ;
+	string event_number ;
+		string event_number:_FillValue = "nan" ;
+	string profile ;
+		string profile:_FillValue = "nan" ;
```

Comment:
string event_number ;  ODF EVENT_HEADER,  EVENT_NUMBER='003'
string profile ;  not sure what is meant by this?

Reply:
Should this information be added from 
the ODF EVENT_HEADER;
  SOUNDING=67.000000,
  DEPTH_OFF_BOTTOM=4.000000,
  MAX_DEPTH=63.000000,

EVENT_HEADER,
  START_DATE_TIME='15-SEP-2018 16:35:00.00',
  END_DATE_TIME='15-SEP-2018 16:44:46.00',